--- origsrc/gegl-0.1.2/gegl/buffer/gegl-buffer.c	2010-02-07 09:31:58.000000000 -0600
+++ src/gegl-0.1.2/gegl/buffer/gegl-buffer.c	2010-07-01 00:56:33.807039100 -0500
@@ -80,7 +80,7 @@
 /* #define GEGL_BUFFER_DEBUG_ALLOCATIONS to print allocation stack
  * traces for leaked GeglBuffers using GNU C libs backtrace_symbols()
  */
-#ifndef G_OS_WIN32
+#ifndef G_PLATFORM_WIN32
 #include <execinfo.h>
 #endif
 
@@ -876,7 +876,7 @@ static gchar *
 gegl_buffer_get_alloc_stack (void)
 {
   char  *result         = NULL;
-#ifdef G_OS_WIN32
+#ifdef G_PLATFORM_WIN32
   result = g_strdup ("backtrack not available on win32\n");
 #else
   void  *functions[MAX_N_FUNCTIONS];
